apps/client/src/pages/ConclusionPage.js

```diff
@@ -86,6 +105,33 @@ const ConclusionPage = ({ topic, checker }) => {
           </Fragment>
         ))}
 
+        {!isMobile && (
+          <DownloadButton
+            type="button"
+            color="primary"
+            onClick={handleDownloadButton}
+            style={{ alignSelf: "flex-start" }}
+          >
+            Conclusie opslaan
+          </DownloadButton>
+        )}
+
+        <PrintOnly style={{ marginTop: 20 }} withBorder>
+          <Alert>
+            <Heading forwardedAs="h2">Proclaimer</Heading>
```

Comment:
Do people know what a Proclaimer is? Can't we just name it 'Let op'?

Reply:
Done